bootstrapping-lambda/fileTransformer.js

```diff
@@ -0,0 +1,5 @@
+const baseTransformer = require("../fileTransformer.js");
+
+module.exports = {
+  ...baseTransformer,
+};
```

Comment:
other modules use a symlink for this purpose, e.g. https://github.com/guardian/pinboard/blob/main/client/fileTransformer.js

might be simpler for now, although we might want to take a broader look at how testing is configured for the project